src/test/java/dk/sdu/se_f22/sharedlibrary/SearchHitsTest.java

```diff
@@ -6,30 +6,35 @@
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
+import dk.sdu.se_f22.productmodule.management.ProductAttribute;
+import dk.sdu.se_f22.sharedlibrary.models.Brand;
+import dk.sdu.se_f22.sharedlibrary.models.Content;
+import dk.sdu.se_f22.sharedlibrary.models.Product;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 import java.util.ArrayList;
 import java.util.Collection;
 
 class SearchHitsTest {
     private SearchHits sh;
-    private Collection products;
-    private Collection brands;
-    private Collection contents;
+    private Collection<Product> products;
+    private Collection<Brand> brands;
+    private Collection<Content> contents;
```

Comment:
This could be changed to a `List<>`, since search module infrastructure requires it from the indexing modules.

Reply:
We could go over to using a List implementation, but our vision was to have a very open type, to enable changing of the specific collection by any group if they wanted.